hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/configuration/DeltaConfig.java

```diff
@@ -88,6 +88,10 @@ public Configuration getConfiguration() {
     private static String REINIT_CONTEXT = "reinitialize_context";
     private static String START_PARTITION = "start_partition";
     private static String DELETE_INPUT_DATA = "delete_input_data";
+    private static String VALIDATE_HIVE = "validate_hive";
+    private static String EXECUTE_ITR_COUNT = "execute_itr_count";
```

Comment:
Can we just rename this to REPEAT_COUNT and use it that way (if that's the intention) ?

Reply:
@n3nash : this refers to the iteration count/number where this node needs to be executed. Basically "execute at iteration count" config. I couldn't come up w/ a better name. If you have good suggestions, let me know.

Reply:
may be "itr_count_to_execute" to be explicit.